lib/explorer/data_frame.ex

```diff
@@ -4636,7 +4636,7 @@ defmodule Explorer.DataFrame do
 
   # Two table verbs
 
-  @valid_join_types [:inner, :left, :right, :outer, :cross]
+  @valid_join_types [:inner, :left, :right, :outer, :outer_coalesce, :cross]
```

Comment:
I wouldn't support outer_coalesce. It is very Polars specific and I am not sure there are other use cases for them besides backwards compatibility.

Reply:
Ok, should we remove it ?

Reply:
Yes, I would not include outer_coalesce at all. :) We can remove it.